src/studio/nuxeo-studio/nuxeo-studio-release-notes/2017-10-02-studio-70.md

```diff
@@ -0,0 +1,49 @@
+---
+title: October 2, 2017
+description: This version polished autocompletion in automation scripting, and improves flow between Modeler and Designer.
+tree_item_index: 999999995
+review:
+    comment: ''
+    date: '2017-10-02'
+    status: ok
+toc: true
+---
+
+{{! multiexcerpt name='summary-v70'}}
+This version improves flow between Designer and Modeler, polishes autocompletion in automation scripting and brings bug fixes.
+{{! /multiexcerpt}}
+
+## Improved Flow Between Studio Modeler and Designer
+One of the frequent feedbacks we received is that you want to be able to switch faster from Modeler to Designer (and vice versa of course).
+
+If you fall in that category, you will be glad to learn that buttons will now appear where appropriate to let you switch from one to the other, directly to the corresponding screen. When creating a document type for instance, configuring its layouts is now one click away.
```

Comment:
_Where appropriate, buttons linking directly to the corresponding screen are now available, making the switch between Modeler and Designer seamless. For example, when creating a document type, configuring its layouts is now one click away._